lib/hmmlearn/hmm.py

```diff
@@ -1049,3 +1050,148 @@ def outer_f(x):  # Outer product over features.
                 c_d = (stats['post_sum'] + nm + nus + nf + 1)[:, None, None]
 
             self.covars_ = c_n / c_d
+
+
+class PoissonHMM(BaseHMM):
+    """
+    Hidden Markov Model with Poisson emissions.
+
+    Attributes
+    ----------
+    monitor_ : ConvergenceMonitor
+        Monitor object used to check the convergence of EM.
+
+    startprob_ : array, shape (n_components, )
+        Initial state occupation distribution.
+
+    transmat_ : array, shape (n_components, n_components)
+        Matrix of transition probabilities between states.
+
+    lambdas_ : array, shape (n_components, n_features)
+        The expectation value of the waiting time parameters for each
+        feature in a given state.
+    """
+
+    def __init__(self, n_components=1, lambda_prior=1.0,
```

Comment:
should be `lambdas_prior` (plural), consistently with `means_prior`, for example.